.changeset/loud-countries-tease.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sap-ux/axios-extension": major
```

Comment:
We still have a `0.x` version, so a `minor` version upgrade is sufficient for the change. So, no `1.0` version here please

Reply:
Changed back to minor

Reply:
thx